samtools.wdl

```diff
@@ -307,6 +307,8 @@ task View {
         File inFile
         File? referenceFasta
         String outputFileName = "view.bam"
+	    Boolean? includeHeader
+        Boolean? outputBam
         Boolean? uncompressedBamOutput
```

Comment:
Boolean shouldn't really be optional, just give them a default value of `false`:
```suggestion
        Boolean uncompressedBamOutput = false
```